docs/glossary.rst

```diff
@@ -0,0 +1,388 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This Reference article is a glossary. It is meant to provide a general overview
+of terms common to CrateDB Cloud. Because it forms a counterpart to the
+Glossary for CrateDB, we mainly cover terms here that are specific or primary
+to CrateDB Cloud; terms particular to the whole CrateDB architecture will
+appear in the CrateDB Glossary instead. We hope this Glossary will help you
+understand the basic meaning of concepts in the Console, Documentation, the
+CLI, and other Cloud-related sources. It is not meant to be a general guide to
+IT or IIoT terminology.
+
+In entries, terms used that have entries of their own are cross-referenced
+through links. The entries are presented in alphabetic order.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _glossary-audit-log:
+
+Audit log
+---------
+
+The Audit Log registers and displays all operations associated with a
+particular :ref:`organization<glossary-org>`. This includes operations on
+:ref:`users<glossary-user>`, on :ref:`clusters<glossary-cluster>`, on
+:ref:`projects<glossary-project>`, and on :ref:`consumers<glossary-consumer>`.
+The Audit Log can be found in the rightmost tab of the Organization overview
+page in the CrateDB Cloud :ref:`Console<glossary-console>`. Only an
+*organization admin* has access to the Audit Log.
+
+
+.. _glossary-aws:
+
+AWS
+---
+
+AWS stands for Amazon Web Services. It is a subsidiary of Amazon that provides
+cloud hosting services. In the near future, CrateDB Cloud will be available via
+AWS.
+
+
+.. _glossary-azure-ad:
+
+Azure AD
+--------
+
+Azure AD (Active Directory) is Microsoft's authentication and sign-in service
+for accessing Microsoft hosted services. CrateDB Cloud uses AzureAD as one of
+the means of sign-up and authentication for its service. For documentation on
+Azure AD, refer to the |Microsoft Documentation|.
+
+
+.. _glossary-CLI:
+
+CLI
+---
+
+CLI stands for Command Line Interface. A Command Line Interface allows a
+customer to use a command line terminal to interact with an application or
+service, in this case CrateDB Cloud. The CLI for CrateDB Cloud is called
+:ref:`Croud<glossary-croud>`.
```

Comment:
I'm not sure who the intended audience is for this glossary, but I am wondering whether we can even omit definitions for AWS and/or CLI.  

These terms don't seem specific to CrateDB Cloud and are also widely known.

Reply:
Yeah that's a fair point. Maybe steers too close to general IT terms.

Reply:
ooh. just to add a third opinion: I think it's fine to keep AWS and CLI

here's my rationale:

- AWS and CLI are both likely to be used in our documents
- when AWS and CLI are used, I would always expand them in the first instance, e.g., "we use Amazon Web Services (AWS) for the next section" (see https://oreillymedia.github.io/production-resources/styleguide/#abbreviationsacronyms)
- any abbreviation or acronym that we would both use in a document and expand for the reader ought to be in the glossary

---

minor: I think we can reduce some of the redundancy/repetition in the text by styling it like this:

```
CLI
---

A Command Line Interface (CLI) allows a customer to use a command-line terminal to interact with an application or service, in this case, CrateDB Cloud.
```

this follows our general style (https://oreillymedia.github.io/production-resources/styleguide/#abbreviationsacronyms) for expanding abbreviations and acronyms and avoids the need to have an additional sentence saying "X stands for Y".

Reply:
Hmm. On the whole, I'm inclined to err on the side of including more, rather than less, stuff in the Glossary. That will make it more useful, even if there are things in it that are potentially redundant. There is a reasonable question: would anyone end up on that page on our documentation without already knowing what a CLI is? On the other hand, CrateDB Cloud is meant to also be easy to use for less technically advanced users, certainly compared to CrateDB. All this being equal, I'd be inclined to leave it in. What do you think, @infoverload ?

Reply:
Based on our conversation today on the Tech Writing Sync..... I think we should remove it!